core.cc

```diff
@@ -30,7 +30,7 @@ class Worker: public Nan::AsyncWorker {
         void HandleOKCallback() {
             Nan::HandleScope scope;
 
-            if(!callback->IsEmpty())
+            if(callback->GetFunction()->IsFunction())
                 callback->Call(0, NULL);
         };
```

Comment:
You can just omit HandleOKCallback, `callback->Call(0, NULL)` is the default implementation.